pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "spack-bot"
+version = "0.0.1"
```

Comment:
Are we planning on "releases" for spack-bot? I was assuming this would be updated live any time a commit was merged.

Reply:
Wouldn't mind doing this eventually but I don't think it's completely necessary for attempt number 1.